cms/djangoapps/contentstore/views/course.py

```diff
@@ -1229,21 +1243,21 @@ def get_user_partition(self):
     @staticmethod
     def get_usage_info(course, store):
         """
-        Get usage information for all Group Configurations.
+        Get usage information for all Group Configurations currently referenced by a split_test instance.
         """
         split_tests = store.get_items(course.id, qualifiers={'category': 'split_test'})
         return GroupConfiguration._get_usage_info(store, course, split_tests)
 
     @staticmethod
-    def add_usage_info(course, store):
+    def get_split_test_partitions_with_usage(course, store):
         """
-        Add usage information to group configurations jsons in course.
+        Add usage information to group configurations jsons that can be used by split_test.
```

Comment:
I'd change this to something that doesn't make it sound like this is a destructive operation. In fact, the 'returns' string below is probably all that is needed.